### ISSUE TITLE
Recent td-to-th change #57883 (cart.php) altered CSS applied to cart details for add-ons and more

### ISSUE THREAD

**Issue by eja2025** [woothemes/woocommerce#59951]
### Prerequisites

- [x] I have carried out troubleshooting steps and I believe I have found a bug.
- [x] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

In cart.php, #57883 changed the ``<td>`` for "product-name" to ``<th>``, causing changes in CSS applied.  So now, for example, if you have add-ons shown in the order details, everything might be in bold, and the quantity (input-multiplier) values might be on the next line.  But the problem doesn't require add-ons;  at a minimum, any product might be bold, suddenly.

While the PR said this td-to-th change was intentional, it doesn't seem like it, since the other td cells were not changed to th, as well.

Temporary workaround is to add the pre-changed version of cart.php to the theme.

### Expected behavior

```
MyProduct
A:  5
B:  1
```

### Actual behavior

(and imagine in bold):
```
MyProduct
A:
   5
B:
   1
```

### Steps to reproduce

1.  Use product with add-ons or variation or really anything, since the issue affects any order details.
2.  Add to cart, and view cart.

### WordPress Environment

Happens on any environment.

### Isolating the problem

- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/products/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Comment by PanosSynetos** [comment]
Thanks for reporting this - 

For @opr - this affects the shortcode-based cart, so please test with Storefront. 

<img width="1659" height="1594" alt="Image" src="https://github.com/user-attachments/assets/cc0c8dd6-035b-4593-b094-7ed9202f0256" />

I believe that we shouldn't have a `th` inside the `tbody` . `th` should exist only inside `thead`

I've assigned this to me, and will fire a PR shortly
